### PR TITLE
Word wrap for settings headers

### PR DESCRIPTION
**Pull request by VladWinner**
Re-opened #39

### CODE REVIEW

**Review by msfjarvis — APPROVED** [review]
Thanks

File: ui/src/main/res/xml/app_restrictions.xml [hunk 1]
```diff
@@ -9,5 +9,6 @@
         android:description="@string/disable_config_export_description"
         android:key="disable_config_export"
         android:restrictionType="bool"
+        android:singleLineTitle="false"
```
Comment (zx2c4) [inline ui/src/main/res/xml/app_restrictions.xml:12]: Does this make sense here in this context? Is this attribute respected by any GUI that shows this item?

**Comment by zx2c4** [comment]
Can you post a screenshot or something of what this accomplishes?

File: ui/src/main/res/xml/app_restrictions.xml (hunk 1 above)
Comment (VladWinner) [reply ui/src/main/res/xml/app_restrictions.xml:12]: I'm not sure because I didn't find "disable_config_export_title" location when using the app

**Comment by VladWinner** [comment]
> Can you post a screenshot or something of what this accomplishes?

https://github.com/WireGuard/wireguard-android/pull/39#issuecomment-1073253749

**Comment by zx2c4** [comment]
https://github.com/WireGuard/wireguard-android/commit/866fd35df5943d0ed3fe4e78120b1c2efecc4b97

**Pull request closed without merging**